Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -961,7 +961,7 @@ airportPickupView push state directionText =
         , id $ getNewIDWithTag "WalkingLottie"
         , margin $ MarginRight 4
         , onAnimationEnd (\action -> do
-          void $ pure $ JB.startLottieProcess JB.lottieAnimationConfig {speed = 1.0, rawJson = (getAssetsBaseUrl FunctionCall) <> "lottie/walk_anim.json", lottieId = getNewIDWithTag "WalkingLottie", repeat = true}
+          void $ pure $ JB.startLottieProcess JB.lottieAnimationConfig {speed = 1.0, rawJson = "https://assets.moving.tech/beckn/common/user/lottie/walk_anim.json", lottieId = getNewIDWithTag "WalkingLottie", repeat = true}
```

Comment:
is it required to hard-code here?

Reply:
Currently there is no function which can fetch `beckn/common/user/` directory, That's why I added it.
It will still first check in local because `startLottieProcess` function extract file name from link and then check in local, If not there then use it from link